tests/approaches/test_nsrt_learning_approach.py

```diff
@@ -97,6 +99,24 @@ def test_nsrt_learning_approach():
                    try_solving=False,
                    sampler_learner="random",
                    learn_side_predicates=True)
+    _test_approach(env_name="repeated_nextto",
+                   approach_name="nsrt_learning",
+                   try_solving=False,
+                   sampler_learner="random",
+                   learn_side_predicates=True,
+                   sidelining_approach="preserve_skeletons")
+
+
+def test_unknown_sidelining_approach():
+    """Test that arbitrary sidelining approach throws error."""
+    with pytest.raises(Exception) as e:
```

Comment:
sort of weird to catch a general Exception here -- since you're raising a ValueError in your code, do `pytest.raises(ValueError)` here